src/newLaunch/baseStage.scss

```diff
@@ -864,3 +864,26 @@
     }
   }
 }
+
+@media screen and (max-width: 690px) {
+  .distribRow {
+    position: relative;
+    margin-top: 25px;
+  }
+
+  .deleteLink {
+    position: absolute;
```

Comment:
Please do not use `position: absolute`.  This is given in the coding guidelines.